src/utils/CoreModule/index.ts

```diff
@@ -18,11 +18,11 @@ import {
 /*--------------------------------- Utility Types ---------------------------------*/
 
 /** Different core modules */
-export type CoreModuleName
-  = 'farmer'
-  | 'milk'
-  | 'loan'
-  ;
+export enum CoreModule {
+  FARMER = 'farmer',
+  MILK = 'milk',
+  LOAN = 'loan',
+}
```

Comment:
are we going to be setup to move info for loans or is this just a place holder for now?

Reply:
We won't be setup yet no, maybe for the time being you're right I should probably remove it since the reducer/thunks don't exist